public/app/ui/components/collections/library/list-result.html

```diff
@@ -0,0 +1,4 @@
+<div class="list-result list-result-library">
+    <span class="pull-left" data-bind="component: displayFor('MediaThumbnail', result)"></span>
+    <a href="#" data-bind="attr: { href: detailUrlFor(result) }, component: displayForLabelField(result)"></a>
```

Comment:
Yeah these views are only placeholders for @mostro20 to fill in the actual fields and layout he wants.  The details views have all the fields listed but need layout.